locales/en/yourloops.json

```diff
@@ -21,6 +21,7 @@
   "admin": "Admin",
   "alert-change-password-email-failed": "Impossible to send the change password e-mail. Please try again later.",
   "alert-change-password-email-success": "E-mail sent successfully",
+  "alert-inactive-user-logged-out": "Your session has expired due to inactivity. Please login again.",
   "alert-hypoglycemic": "Severe hypoglycemia",
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏
```suggestion
  "alert-hypoglycemic": "Severe hypoglycemia",
  "alert-inactive-user-logged-out": "Your session has expired due to inactivity. Please login again.",
```